rotkehlchen/db/schema.py

```diff
@@ -659,6 +659,16 @@
 );
 """
 
+DB_CREATE_ACCOUNTING_RULE_EVENTS = """
+CREATE TABLE IF NOT EXISTS accounting_rule_events(
+    identifier INTEGER NOT NULL PRIMARY KEY,
+    rule_id INTEGER NOT NULL,
+    event_id INTEGER NOT NULL,
```

Comment:
shouldn't this also have a Foreign key relation to the event ID itself? Otherwise we have dangling rules.